base/range.jl

```diff
@@ -157,6 +170,8 @@ type can represent values smaller than `oneunit(Float64)`.
 """
 abstract type OrdinalRange{T,S} <: AbstractRange{T} end
 
+convert(::Type{OrdinalRange{T,S}}, r::OrdinalRange{T,S}) where {T,S} = r
```

Comment:
Not strictly necessary for this PR, I think, but I noticed their lack and thought it might be good for disambiguating.